deployment.yml

```diff
@@ -23,11 +23,20 @@ spec:
             valueFrom:
               secretKeyRef:
                 name: self-hosted-runner-creds
-                key: TOKEN
+                key: GITHUB_TOKEN
```

Comment:
Could we continue to use `TOKEN` here to differentiate the PAT from the [automatically-provisioned `GITHUB_TOKEN`](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token)? Or maybe another name?